packages/nodes-design-tokens/src/nodes/invert.ts

```diff
@@ -32,15 +32,15 @@ export default class InvertNode extends Node {
 
 	execute(): void | Promise<void> {
 		const value = this.inputs.tokens.value;
+		const reversedValues = [...value].reverse();
 
 		const inverted = value.map((x, i) => {
-			const vals = inverted[inverted.length - i - 1];
```

Comment:
Hah well at least you can say this idea was kinda creative/clever in theory

Reply:
:))